tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -1186,14 +1186,14 @@ static PyObject* THPVariable_set_(
         " for argument 1 'storage'");
       auto dispatch_set_ = [](const Tensor& self, Storage source) -> Tensor {
         pybind11::gil_scoped_release no_gil;
-        return self.set_(source);
+        return self.set_(std::move(source));
       };
       return wrap(dispatch_set_(self, storage));
```

Comment:
Actually, should any of these wrap calls also move their args? Might be good for future PR.

Reply:
There will be 3,4,5... until all are fixed.

Reply:
Most importantly, it should be included into linterruner